.docksal/docksal.env

```diff
@@ -1,28 +1,23 @@
-# This is a shared configuration file that is intended to be stored in the project repo.
```

Comment:
I would not expect users to be mindful of the docs. This copy explains a fundamental different between `docksal.env` and `docksal-local.env`. Let's keep it.

Reply:
```suggestion
# This is a shared configuration file that is intended to be stored in the project repo.
```